### PR TITLE
Major design update 2019

### PR DESCRIPTION
**Pull request by RobinRSchulz**
* Adjust design to pipeline config history's design.

### CODE REVIEW

File: src/main/resources/hudson/plugins/jobConfigHistory/JobConfigHistoryRootAction/showDiffFiles.jelly [hunk 1]
```diff
@@ -1,6 +1,15 @@
...
-  <l:layout title="Job Configuration History" css="/plugin/jobConfigHistory/diff_highlight.css">
+  <l:layout title="Job Configuration History">
+
+    <link rel="stylesheet" type="text/css" href="${rootURL}/plugin/jobConfigHistory/css/diff_highlight.css"></link>
+    <link rel="stylesheet" type="text/css" href="${rootURL}/plugin/jobConfigHistory/css/style.css"></link>
+
+    <!--syntax highlighting-->
+    <link rel="stylesheet" type="text/css" href="${rootURL}/plugin/jobConfigHistory/highlight.js/styles/purebasic.css"/>
+    <script src="${rootURL}/plugin/jobConfigHistory/highlight.js/highlight.pack.js"></script>
+    <script>hljs.initHighlightingOnLoad();</script>
```
Comment (stefanbrausch) [inline src/main/resources/hudson/plugins/jobConfigHistory/JobConfigHistoryRootAction/showDiffFiles.jelly]: ; ?

File: src/main/resources/hudson/plugins/jobConfigHistory/JobConfigHistoryRootAction/restoreQuestion.jelly [hunk 2]
```diff
@@ -1,15 +1,34 @@
...
-      <f:form action="restore?name=${name}" name="restore" method="post">
-        <f:entry>
-          <f:submit value="${%Yes}"/>
-        </f:entry>
-      </f:form>
+
+      <div style="text-align:center">
+        <div class="jch-warning">
+          <div style="float:left; font-size:30px; font-weight:bold">
+            ⚠️
```
Comment (stefanbrausch) [inline src/main/resources/hudson/plugins/jobConfigHistory/JobConfigHistoryRootAction/restoreQuestion.jelly]: check item encoding

File: src/main/resources/hudson/plugins/jobConfigHistory/JobConfigHistoryRootAction/index.jelly [hunk 3]
```diff
@@ -50,87 +79,100 @@
...
-                          <j:if test="${it.hasConfigurePermission() || it.hasJobConfigurePermission()}">
-                            <!--this prevents people from destroying their history by trying to restore deleted folder jobs
-                             TODO Fix that. (see code)-->
-                            <j:if test="${config.job.replace('/jobs', '/job').equals(config.job)}">
-                              <a id="restore" href="restoreQuestion?name=${config.getJob()}">
-                                <img src="${resURL}/plugin/jobConfigHistory/img/restore.png" alt="${%Restore}" />
-                              </a>
+                        <!-- TODO check whether job has a JobConfigHistoryProjectAction before hyperlinking -->
+                        <a href="${it.createLinkToFiles(config,'xml')}">
+                          <img src="${imagesURL}/24x24/document.png" alt="" />${%View as XML}
```
Comment (stefanbrausch) [inline src/main/resources/hudson/plugins/jobConfigHistory/JobConfigHistoryRootAction/index.jelly]: remove alt or fill it

File: src/main/resources/hudson/plugins/jobConfigHistory/JobConfigHistoryProjectAction/showDiffFiles.jelly [hunk 4]
```diff
@@ -2,7 +2,16 @@
...
-  <l:layout title="Job Configuration History" css="/plugin/jobConfigHistory/diff_highlight.css">
+  <l:layout title="Job Configuration History">
+
+    <link rel="stylesheet" type="text/css" href="${rootURL}/plugin/jobConfigHistory/css/diff_highlight.css"></link>
+    <link rel="stylesheet" type="text/css" href="${rootURL}/plugin/jobConfigHistory/css/style.css"></link>
+
+    <!--syntax highlighting-->
+    <link rel="stylesheet" type="text/css" href="${rootURL}/plugin/jobConfigHistory/highlight.js/styles/purebasic.css"/>
+    <script src="${rootURL}/plugin/jobConfigHistory/highlight.js/highlight.pack.js"></script>
+    <script>hljs.initHighlightingOnLoad();</script>
```
Comment (stefanbrausch) [inline src/main/resources/hudson/plugins/jobConfigHistory/JobConfigHistoryProjectAction/showDiffFiles.jelly]: ; ?

**Comment by stefanbrausch** [comment]
@RobinRSchulz  Please add license declaration for highlight.js

File: src/main/resources/hudson/plugins/jobConfigHistory/JobConfigHistoryRootAction/restoreQuestion.jelly (hunk 2 above)
Comment (RobinRSchulz) [reply src/main/resources/hudson/plugins/jobConfigHistory/JobConfigHistoryRootAction/restoreQuestion.jelly]: done.

File: src/main/resources/hudson/plugins/jobConfigHistory/JobConfigHistoryRootAction/index.jelly (hunk 3 above)
Comment (RobinRSchulz) [reply src/main/resources/hudson/plugins/jobConfigHistory/JobConfigHistoryRootAction/index.jelly]: done

**Pull request merged**